docs/static/monitoring-apis.asciidoc

```diff
@@ -220,47 +444,78 @@ GET /_node/hot_threads
+          "org.jruby.internal.runtime.NativeThread.join(NativeThread.java:75)",
+          "org.jruby.RubyThread.join(RubyThread.java:697)",
+          "org.jruby.RubyThread$INVOKER$i$0$1$join.call(RubyThread$INVOKER$i$0$1$join.gen)",
+          "org.jruby.internal.runtime.methods.JavaMethod$JavaMethodN.call(JavaMethod.java:663)",
+          "org.jruby.internal.runtime.methods.DynamicMethod.call(DynamicMethod.java:198)",
+          "org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:306)",
+          "org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:136)",
+          "org.jruby.ast.CallNoArgNode.interpret(CallNoArgNode.java:60)"
+        ]
+      },
+      {
+        "name": "Ruby-0-Thread-17",
+        "percent_of_cpu_time": 0.05,
+        "state": "timed_waiting",
+        "path": "/Users/dedemorton/BuildTesting/5.0.0-alpha4/logstash-5.0.0-alpha4/logstash-core/lib/logstash/pipeline.rb:467",
+        "traces": [
+          "java.lang.Object.wait(Native Method)",
+          "org.jruby.RubyThread.sleep(RubyThread.java:1002)",
+          "org.jruby.RubyKernel.sleep(RubyKernel.java:803)",
+          "org.jruby.RubyKernel$INVOKER$s$0$1$sleep.call(RubyKernel$INVOKER$s$0$1$sleep.gen)",
+          "org.jruby.internal.runtime.methods.JavaMethod$JavaMethodN.call(JavaMethod.java:667)",
+          "org.jruby.internal.runtime.methods.DynamicMethod.call(DynamicMethod.java:206)",
+          "org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:168)",
+          "rubyjit.Module$$stoppable_sleep_c19c1639527ca7d373b5093f339d26538f1c21ef1028566121.__file__(/Users/dedemorton/BuildTesting/5.0.0-alpha4/logstash-5.0.0-alpha4/vendor/bundle/jruby/1.9/gems/stud-0.0.22/lib/stud/interval.rb:84)",
+          "rubyjit.Module$$stoppable_sleep_c19c1639527ca7d373b5093f339d26538f1c21ef1028566121.__file__(/Users/dedemorton/BuildTesting/5.0.0-alpha4/logstash-5.0.0-alpha4/vendor/bundle/jruby/1.9/gems/stud-0.0.22/lib/stud/interval.rb)",
+          "org.jruby.ast.executable.AbstractScript.__file__(AbstractScript.java:46)"
+        ]
+      }
+    ]
+  }
+}
 --------------------------------------------------
 
+//REVIEWERS: Curious why the hot threads API is the only one that (appears) to accept parameters. Am I missing parameters in the other APIs?
+
+The parameters allowed are:
+
+[horizontal]
+`threads`:: 	        The number of hot threads to return. The default is 3. 
+`human`:: 	            If true, returns plain text instead of JSON format. The default is false. 
+`ignore_idle_threads`:: If true, does not return idle threads. The default is true.
+
 You can use the `?human` parameter to return the document in a human-readable format.
 
 [source,js]
 --------------------------------------------------
 GET /_node/hot_threads?human=true
 --------------------------------------------------
 
+//REVIEWERS: Does the ?human parameter still work? It didn't work in the build that I used.
```

Comment:
@suyograo I created an https://github.com/elastic/logstash/issues/5563 that describes the behavior that I'm seeing in Alpha4 (which is different than Alpha3). If the behavior is intentional, please close the issue. It just doesn't seem quite right to me.